### ISSUE TITLE
make sure /usr/local/include/ exists as a directory when installing

### ISSUE THREAD

**Issue by human-blip** [g0orx/wdsp#12]
The first time I built and installed wdsp I got caught by that fact that - for some reason - /usr/local/include was not present so the copy of wdsp.h didn't work as expected.  The thing is I didn't notice until I tried to build linhdspr and /usr/local/include wdspr.h could not be found.

diff --git a/Makefile b/Makefile
index 037ef82..6a8271a 100644
--- a/Makefile
+++ b/Makefile
@@ -222,12 +222,13 @@ $(JAVA_PROGRAM): $(JAVA_OBJS)
        $(COMPILE) $(OPTIONS) $(GTKOPTIONS) -c -o $@ $<
 
 install: $(PROGRAM)
-       cp wdsp.h /usr/local/include
-       cp $(PROGRAM) /usr/local/lib
+       mkdir -p /usr/local/include/
+       cp wdsp.h /usr/local/include/
+       cp $(PROGRAM) /usr/local/lib/
        ldconfig
 
 install_java: $(JAVA_PROGRAM)
-       cp $(JAVA_PROGRAM) /usr/local/lib
+       cp $(JAVA_PROGRAM) /usr/local/lib/
 
 release: $(PROGRAM)
        cp $(PROGRAM) ../pihpsdr.src/release/pihpsdr

**Comment by g0orx** [comment]
What Linux distribution are you using?

**Comment by human-blip** [comment]
debian buster

**Comment by g0orx** [comment]
I have tried the latest Debian Buster install and it does have /usr/local/include, /usr/local/lib and /usr/local/bin, so not sure why you had this problem.  I am not aware of any distribution that does not have /usr/local and it's sub directories.

**Comment by human-blip** [comment]
Yes it took me by surprise as well. Not sure how it happened. In any case adding the trailing / to the cp would force an error if the directory is not present rather than failing silently.
